misc/e2elive.py

```diff
@@ -82,8 +82,8 @@ def main():
         import boto3
         from botocore.config import Config
         from botocore import UNSIGNED
-
         s3 = boto3.client("s3", config=Config(signature_version=UNSIGNED))
+        s3 = boto3.client('s3', config=Config(signature_version=UNSIGNED))
```

Comment:
Merge conflict with the python formatting, maybe?
```suggestion
        s3 = boto3.client("s3", config=Config(signature_version=UNSIGNED))
```

Reply:
Yeah I guess I messed up rebasing off of develop.